addons/api/tests/unit/serializers/storage-bucket-test.js

```diff
@@ -161,6 +154,57 @@ module('Unit | Serializer | storage bucket', function (hooks) {
     assert.deepEqual(serializedRecord, expectedResult);
   });
 
+  test('it serializes when updating a storage bucket with dynamic credential correctly', async function (assert) {
```

Comment:
Can we add values for the `access_key_id` and `secret_access_key` to confirm they properly get set to null?